benchmarks/run_benchmark.sh

```diff
@@ -0,0 +1,222 @@
+#! /bin/bash
+set -eo pipefail
+
+choice=${1}
```

Comment:
Could you print out Usage help messages when user doesn't provide any parameters?